sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/data_version.py

```diff
@@ -0,0 +1,39 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict
+from azure.ai.ml._schema._deployment.batch.batch_job_property import OutputDataSchema
+from azure.ai.ml._schema.core.fields import ArmStr
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+
+
+class DataVersion:
```

Comment:
DataVersion and ComputeBinding are both legacy concepts. I'm hessitant to expose these to customers. What's the plan to refactor BatchJob to match other job structures?